hack/make/.build-deb/control

```diff
@@ -13,7 +13,7 @@ Recommends: aufs-tools,
             git,
             xz-utils,
             ${apparmor:Recommends}
-Conflicts: docker (<< 1.5~), docker.io, lxc-docker, lxc-docker-virtual-package
+Conflicts: docker (<< 1.5~), docker.io, lxc-docker, lxc-docker-virtual-package, docker-engine-cs
```

Comment:
IMO, this shouldn't be listed as a conflict in the open source package, but in the `docker-engine-cs` package.